app/views/search/_results.html.slim

```diff
@@ -0,0 +1,27 @@
+.col-xs-12.col-sm-9
+  .page-details__row--quarter
+    = form_tag search_path, method: :get do
+      .row
+        .col-xs-12.col-sm-6
+          h6
+            span= "Results for \"#{@query}\" "
+            span.gray-light
+              | &bull;
+              = " #{number_with_delimiter(@total_count, delimiter: ' ')}"
```

Comment:
Should I also add `results` after? I'm not sure because we already have `Results for` right before sooo, let me know ;)